core/components/atoms/dropdown/errorTemplate.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Text } from '@/index';
+
+interface ErrorTemplateProps {
+  dropdownStyle: React.CSSProperties;
+  width: number | string | undefined;
+  errorType: string;
+}
+
+const errorTemplateHeader: Record<string, string> = {
```

Comment:
This variable name should be `errorTemplateTitle`

Reply:
Done